### DISCUSSION TITLE
some good injector idea ?

### DISCUSSION THREAD

**Discussion by rohlicekk** [danielkrupinski/Osiris#3419] [Q&A]
please

**Comment by tusharborkar360** [comment]
Go for Process Hacker 2. It's like something whitelisted by Valve.
paste the Osiris DLL in Windows folder.

**Reply by diekunstderfuge** [reply]
I'm using Sazinjector and it's working. But I miss CSGhost.

**Comment by mdng568** [comment]
I recommend using CSGhost newest version.
